Core/IntegrationTests/Fixtures.cs

```diff
@@ -14,8 +14,8 @@ namespace TestsIntegration
 {
   public static class Fixtures
   {
-    public static readonly ServerInfo Server = new ServerInfo { url = "https://latest.speckle.dev", name = "Test Server" };
-    
+    public static readonly ServerInfo Server = new ServerInfo { url = "http://localhost:3000", name = "Docker Server" };
```

Comment:
Only thing remaining for this issue to be closed is to decide what to do with this hardcoded URL so local testing would still use `latest`.